adal/src/main/java/com/microsoft/aad/adal/AuthenticationException.java

```diff
@@ -33,6 +41,12 @@
 
     private ADALError mCode;
 
+    private Map<String, String> mResponseBody;
```

Comment:
can we initialize both the maps to null, so that in non-http cases we are not returning uninitialized values.